### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by directvox** [ninoguba/ionic-researchKit#38]
I would like to add localisation (i18n) to the library.  What is the best method/framework for doing this.  I was thinking of Angular-translate.  What do you think.

**Comment by ninoguba** [comment]
Yeah I think Angular-Translate will work. Go for it! Looking forward to your contribution
